model/registry/tls.yaml

```diff
@@ -0,0 +1,195 @@
+groups:
+  - id: registry.tls
+    prefix: tls
+    type: attribute_group
+    brief: "This document defines semantic convention attributes in the TLS namespace."
+    attributes:
+      - id: cipher
+        requirement_level: opt_in
```

Comment:
requirement level belongs to attribute usage, not definition, we shouldn't have it in registry

Reply:
+1 good catch!

Reply:
apologies, I missed that! Let me fix that